### ISSUE TITLE
Applying DEDPUL to completely unlabeled test data?

### ISSUE THREAD

**Issue by jaredsagendorf** [dimonenka/DEDPUL#1]
Hello, great method you have developed! I was wondering if you could give advice for applying DEDPUL to test data which is completely unlabelled?

Suppose I have a training set (Xp, Xu) with positive and unlabeled samples. I go through all the steps to train an NTC to learn y(x), and then estimate a* and the posterior probabilities as described in the paper.

Now I have a test set (which we assume is sampled from the same distributions as the training data), however here I have no labels - can I somehow still use the results from the training data to estimate P and N on the test set?

**Comment by dimonenka** [comment]
Hey!

Thanks, I appreciate it!

Yea, what you’re talking about is transductive vs inductive problem formulation, i.e. to only label the data at hand vs to build a general classifier. The latter is currently not covered in the paper nor in the code at the github, but I have the code for that locally. I’ll just add it to github.

**Comment by jaredsagendorf** [comment]
Great thanks, I'll check it out!
